static/webmention.js

```diff
@@ -212,13 +214,8 @@ A more detailed example:
         return text;
     }
 
-    function pluralise(word, count) {
-      // TODO: This pluralising depends on the language but assumes English
-      return t(word) + count > 1 ? 's' : '';
-    }
-
     function formatComments(comments) {
-        var html = '<h2>' + comments.length + pluralise(' Response', comments.length) +
+        var html = '<h2>' + comments.length + t(' Responses') +
```

Comment:
My point was to just like, remove the count entirely, e.g.

```
var html = '<h2>' + t('Responses') + '</h2>';
```

Reply:
Done.
(You might enjoy item 6 of [Adding line comments to a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request))

Reply:
Yeah, I'm aware of that, but I'm also lazy :)